### ISSUE TITLE
When adding a new test format, if templates aren't made, warnings are thrown

### ISSUE THREAD

**Issue by shopblocks** [UnionOfRAD/lithium#1229]
When creating the 'plain' format for the tests, there was a warning thrown part way through the test, which wasn't very clear as to what the issue was.

It  turns out, that in Report.php:237.

```php
include $params['template'];
```

the value of template is null, when the format doesn't exist.

```php
$template = Libraries::locate('test.templates', $template, array(
    'filter' => false, 'type' => 'file', 'suffix' => ".{$config['format']}.php"
));
```

I think this should be made clearer.

One way to do it, would be to add something like this:

```php
if($template === null) {
    throw new TemplateException("Templates for format `{$config['format']}` not found in `test/templates`");
}
```

to the render() method in Report.php, but i'm not sure if there is a better method that you'd prefer.

Thanks.

**Comment by shopblocks** [comment]
Are you happy with there being an exception thrown? If so, i can submit this as a PR.

**Comment by davidpersson** [comment]
This is in test\Report? In this case I'm down with it :)

**Comment by shopblocks** [comment]
Yes, It is a bit weird though. If it hits there, the tests obviously stop reporting anything, and instead simply shows the Exception message, but since it would be caught before CI, i think thats fine.

I'll submit the PR for it.

**Comment by shopblocks** [comment]
PR made https://github.com/UnionOfRAD/lithium/pull/1233

**Comment by davidpersson** [comment]
Fixed by #1233 and merged in c87c87d3ea3f4616d19be17080137fdab6e2b998. Thanks for reporting!
